src/app/page.tsx

```diff
@@ -18,14 +20,16 @@ import WhoWeAre from '@/components/WhoWeAre';
 export default function HomePage() {
   return (
     <main>
-      <div className='relative h-[4000px]'>
+      <div className='relative'>
         <AmberBox />
-        <CyanBoxAndHeader />
+        <HeaderAndMainClaim />
         <div className='flex w-full flex-col items-center justify-center'>
           <WhatWeDo />
           <OurMission />
           <WhoWeAre />
           <Testimonials />
+          <LetsGetInTouch />
+          <Footer text='Imprint' link='/imprint' />
```

Comment:
any reason why we pass the imprint into the foot as params?

Reply:
Yes, I am using the `<Footer/>` component also in `src/app/(header-footer)/layout.tsx` here the text is 'Home' and the link should lead back to the main page.